index.html

```diff
@@ -307,10 +307,10 @@ <h3>History</h3>
                     <div class="feature-description">
                         <p><strong>Go back in time</strong></p>
                         <ul>
-                            <li>Watch your favourites again</li>
-                            <li>Search suggestions based on your last searches</li>
-                            <li>Easily open what you watch most</li>
-                            <li class="text-success">Control what we keep for you</li>
+                            <li>Watch favourites again</li>
+                            <li>Search-suggestions based on your last searches</li>
```

Comment:
```suggestion
                            <li>Search suggestions based on your last searches</li>
```

Reply:
+1

Reply:
It isn't the same sentence anymore if searching _in_ suggestions. The distinction is made.